### PR TITLE
Istio Ingress for Project Jackson

### PR DESCRIPTION
**Pull request by evanlouie**
- Renamed existing manifest to a numbered name schema
- Added Istio Ingress rules to serve Project Jackson from a single endpoint. `/api` serves the api, all other routes are served the UI.

### CODE REVIEW

**Comment by andrebriggs** [comment]
@evanlouie are you guys planning on making a helm chart for the existing API/UI deployments/services?

File: demo/2-istio-services.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Istio Ingress Example for Project Jackson
+#
+# Authors:
```
Comment (andrebriggs) [inline demo/2-istio-services.yaml]: Necessary? VCS like Git make this information redudant

Comment (evanlouie) [reply demo/2-istio-services.yaml]: It's something I grew accustomed to doing for open source projects where files might be moved between repos and git history lost; handy to know who to complain to if something breaks haha

**Comment by evanlouie** [comment]
closes #19

**Comment by andrebriggs** [comment]
@evanlouie any updated instructions here? I applied the istio-service.yaml file and I'm a little unclear if I should be hitting the same same ui-lb-service external endpoint to test (doesn't seem to make sense) or if there is another endpoint one should hit to see single endpoint for ui and api.

**Comment by evanlouie** [comment]
@andrebriggs you have to ping the istio-ingress, if you do a `kubectl get svc -n istio-system` there will be one service with an external ip. Ping that one.

**Pull request merged**